### PR TITLE
Feature palette

### PR DESCRIPTION
**Pull request by wfortin**
* Add typography utility from https://github.com/basscss/utility-typography (we'll need to document that)
* Basic color backgrounds
* Display all is styleguide 

image

### CODE REVIEW

**Comment by pastjean** [comment]
whoohoo :+1:

File: scss/common/palette.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
+$orange: #f58020;
+$white: #ffffff;
+$light-gray: #f5f6f7;
+$light-blue: #2b4b72;
+$dark-blue: #1c2e44;
+$black: #1d1f21;
+
```
Comment (GermainBergeron) [inline scss/common/palette.scss]: Maybe we could/should leverage sass loops & lists
```
$colors: orange $orange, white $white, ...
@each $color in $colors {
  .bg-{nth($color, 1)} {
    background-color: nth($color, 2);
  }
}
```

File: dist/css/CoveoStyleGuide.css [hunk 2]
```diff
@@ -49,6 +49,36 @@ html {
 body {
   font-family: "Open Sans", Arial, Helvetica, sans-serif; }
 
+.bg-orange {
+  background-color: #f58020; }
```
Comment (JPLachance) [inline dist/css/CoveoStyleGuide.css]: Use variables for colors?

File: scss/common/palette.scss (hunk 1 above)
Comment (wfortin) [inline scss/common/palette.scss]: Done !

**Comment by flehoux** [comment]
shipittt

**Comment by JPLachance** [comment]
Ship it!

File: gulpTasks/sass.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ gulp.task('sass', 'Compile sass files to dist folder', ['sprites'], function ()
 });
 
 gulp.task('sass:format', function() {
-    return gulp.src('./scss/**/*.scss')
+    return gulp.src(['./scss/**/*.scss', '!./scss/sprites.scss', '!./scss/common/palette.scss'])
```
Comment (marilynechan) [inline gulpTasks/sass.js]: I don't understand why you're excluding the palette?

**Comment by marilynechan** [comment]
Ship it!

**Pull request merged**